### PR TITLE
Feature/pio inspect2

### PR DESCRIPTION
**Pull request by elyulka**
# TODO

Empty

### CODE REVIEW

File: app/media/styles/modules/inspect.less [hunk 1]
```diff
@@ -0,0 +1,16 @@
+.inspect__top-block__table {
```
Comment (ivankravets) [inline app/media/styles/modules/inspect.less]: Please fix style names

-> inspect-stats-block

File: app/media/styles/modules/inspect.less [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    width: auto;
+}
+
+.inspect__top-block__cell {
+    padding: 1px 5px;
+    white-space: nowrap
+}
+
+.inspect__top-block__cell_size {
+    font-weight: bold;
```
Comment (ivankravets) [inline app/media/styles/modules/inspect.less]: Why do we need ton of code? Just `<b>Hello</b>`

File: app/modules/inspect/containers/configuration.jsx [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import { connect } from 'react-redux';
+import { loadProjects } from '@project/actions';
+import { selectProjects } from '@project/selectors';
+import { selectSavedConfiguration } from '@inspect/selectors';
+
+class InspectionFormComponent extends React.Component {
+  static propTypes = {
```
Comment (ivankravets) [inline app/modules/inspect/containers/configuration.jsx]: Group items in propTypes:
 - objects/data
- callbacks/functions, such as "loadProjects: PropTypes.func.isRequired", etc.

File: app/modules/inspect/containers/configuration.jsx [hunk 4]
```diff
@@ -0,0 +1,245 @@
...
+  componentWillUnmount() {
+    this._isMounted = false;
+  }
+
+  isValid() {
+    const { projectDir, env, memory, code } = this.props.form.getFieldsValue();
+    return projectDir && projectDir.length && env && env.length && (memory || code);
+  }
+
+  handleSubmit = e => {
```
Comment (ivankravets) [inline app/modules/inspect/containers/configuration.jsx]: Why we can't use original function syntax? 99% of this project use NAME(args)  {} but staring with PIO Inspect we have in file mix with "isValid() {" , "handleSubmit = e => {".

Let's use a unified format for all functions.

File: app/modules/inspect/containers/configuration.jsx [hunk 5]
```diff
@@ -0,0 +1,245 @@
...
+    this.setState({ running: true, error: undefined });
+    this.props.saveConfiguration(configuration);
+    this.props.inspectProject(configuration, (_result, error) => {
+      if (this._isMounted) {
+        this.setState({ running: false, error });
+      }
+    });
+  };
+
+  handleProjectChange = projectDir => {
```
Comment (ivankravets) [inline app/modules/inspect/containers/configuration.jsx]: The same,  => `handleProjectChange(projectDir) { `

File: app/modules/inspect/sagas.js [hunk 6]
```diff
@@ -0,0 +1,166 @@
...
+        if (code) {
+          codeCheckResult = yield call(_inspectCode, configuration);
+        }
+      }
+      const entity = { memory: memoryResult, codeCheck: codeCheckResult };
+      yield put(saveConfiguration(configuration));
+      yield put(updateEntity(RESULT_KEY, entity));
+
+      const state = yield select();
+      if (state.router) {
```
Comment (ivankravets) [inline app/modules/inspect/sagas.js]: What will be here? Redirect? onEnd?

Comment (elyulka) [reply app/modules/inspect/sagas.js]: both

**Pull request merged**